### PR TITLE
SOAPDenovo2

### PR DESCRIPTION
**Pull request by jerowe**
* [ ] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by jerowe** [comment]
What's this.

```
Using Anaconda Cloud api site https://api.anaconda.org

Error: Failed to render jinja template in /bioconda-recipes/recipes/boltons/meta.yaml:

'dict object' has no attribute 'GIT_DESCRIBE_TAG'
```

File: recipes/soapdenovo2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+about:
+  home: http://soap.genomics.org.cn/soapdenovo.html
+  license: 'GNU'
```
Comment (johanneskoester) [inline recipes/soapdenovo2/meta.yaml]: Do you mean GPL?

**Comment by johanneskoester** [comment]
Do the soap guys say anything about the system this was build on?

**Comment by jerowe** [comment]
I can't find any info on how its built.

Let me try building it from source. I had trouble in the past, but there's a new version and maybe its all fixed now.

**Comment by jerowe** [comment]
@bgruening , my osx wizard, could you take a look at this?

**Comment by jerowe** [comment]
Also, when do you sleep?

**Comment by bgruening** [comment]
@jerowe you skip the OSX build isn't it?
Despite all rumours I do sleep! :)

Btw. I don't have an OSX at all :)

**Comment by jerowe** [comment]
I was skipping the osx build. I should probably just stay away from work until I've had enough coffee. ;-)

Sleep is wonderful. My youngest only recently started sleeping through the night and I'm still celebrating. ;-)

Even with no osx you still are more knowledgeable than me!

**Comment by jerowe** [comment]
It passed! I was applying linux patches to the osx builds, but apparently the osx will build with no troubles. Who knew!

File: recipes/soapdenovo2/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      - samtools ==0.1.19
+  run:
+      - libgcc # [linux]
+      - zlib
+      - samtools ==0.1.19
+
+source:
+  fn: soapdenovo2-t240.tgz
+  url: https://sourceforge.net/projects/soapdenovo2/files/SOAPdenovo2/src/r240/SOAPdenovo2-src-r240.tgz/download # [linux]
+  url: https://sourceforge.net/projects/soapdenovo2/files/SOAPdenovo2/src/r240/SOAPdenovo2-src-r240-mac.tgz/download # [osx]
```
Comment (bgruening) [inline recipes/soapdenovo2/meta.yaml]: Interesting different source for different OS, why oh why? :)

**Comment by bgruening** [comment]
Thanks a lot @jerowe!

File: recipes/soapdenovo2/meta.yaml (hunk 2 above)
Comment (jerowe) [inline recipes/soapdenovo2/meta.yaml]: I don't know. Because they are there. ;-)

**Comment by jerowe** [comment]
Thanks! Hopefully I'll get soapdenovo2-prepare sorted next!

**Pull request merged**